newspaper/models.py

```diff
@@ -351,25 +347,25 @@ class ArticlePage(RoutablePageMixin, Page):
     featured_on_section = models.CharField(max_length=2, choices=yes_no, blank=True, default='y')
     featured_on_main = models.CharField(max_length=2, choices=yes_no, blank=True, default='y')
 
-    # make sure it displays both the authors' names and their position
-    authors = models.ManyToManyField(AuthorsPage, help_text="Select author names")
-
-    tags = models.CharField(max_length = 255, blank=True)
-
-    # search_fields = Page.search_fields + [
-    #     # Index the human-readable string for searching.
-    #     index.SearchField('authors'),
-
-    # ]
+    tags = models.CharField(max_length=255, blank=True)
 
+    api_fields = [
+        APIField('summary'),
+        APIField('content'),
+        APIField('section'),
+        APIField('featured_on_section'),
+        APIField('featured_on_main'),
+        APIField('tags'),
+        APIField('authors'),
+    ]
 
     content_panels = Page.content_panels + [
         FieldPanel('section', classname='class'),
         FieldPanel('summary', classname='class'),
         FieldPanel('featured_on_section', classname='class'),
         FieldPanel('featured_on_main', classname='class'),
         FieldPanel('tags', classname='full'),
-        FieldPanel('authors', classname='class'),
+        InlinePanel('authors', heading='authors', help_text='Add contributing authors')
```

Comment:
@jaja-t could this be a solution to the authors issue we were talking about?

Reply:
For reference, this is what it looks like in the CMS with this workaround:

<img width="980" alt="screen shot 2019-01-10 at 5 00 19 pm" src="https://user-images.githubusercontent.com/10607620/50999837-4ba49400-14f9-11e9-82ce-58c360a91ae1.png">